src/components/ContactAcceptDialog/ContactAcceptDialog.tsx

```diff
@@ -0,0 +1,78 @@
+import { useQueryClient } from '@tanstack/react-query';
+import { Row } from '@tanstack/react-table';
+import { UserRoundCheck } from 'lucide-react';
+import { useState } from 'react';
+
+import { toast } from 'sonner';
+
+import { Button } from '@/components/ui/button.tsx';
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from '@/components/ui/dialog.tsx';
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from '@/components/ui/tooltip.tsx';
+import { useSpvWalletClient } from '@/contexts';
+import { errorWrapper } from '@/utils';
+
+export interface ContactAcceptDialogProps {
+  row: Row<any>;
+}
+
+export const ContactAcceptDialog = ({ row }: ContactAcceptDialogProps) => {
+  const [isAcceptDialogOpen, setIsAcceptDialogOpen] = useState(false);
+
+  const { spvWalletClient } = useSpvWalletClient();
+
+  const queryClient = useQueryClient();
+
+  const handleAcceptDialogOpen = () => {
+    setIsAcceptDialogOpen((prev) => !prev);
+  };
+
+  const handleAcceptContact = async () => {
+    try {
+      await spvWalletClient?.AdminAcceptContact(row.original.id);
+      await queryClient.invalidateQueries();
+      setIsAcceptDialogOpen(false);
+      toast.success('Contact accepted');
+    } catch (err) {
+      toast.error('Failed accept contact');
```

Comment:
I think a clearer phrase would be: "Failed to accept contact."

Please check other toast messages for correctness — maybe check them with ChatGPT or some other AI.

Reply:
Updated